__tests__/app.test.js

```diff
@@ -484,4 +484,42 @@ describe.only("GET /api/articles with sorting queries", () => {
       expect(msg).toBe("Not available")
     })
   })
-})
\ No newline at end of file
+})
+describe.only("GET /api/articles topic query", () => {
+test("200: retrieves articles belonging to a specific topic value", () => {
+  return request(app)
+  .get("/api/articles?topic=cats")
+  .expect(200)
+  .then(({body}) => {
+    const {articles} = body
+    expect(articles[0].topic).toBe("cats")
+  })
+})
+test("200: retrieves all articles if no endpoint specified", () => {
+  return request(app)
+  .get("/api/articles")
```

Comment:
This has previously been tested for when you did the initial tests for the endpoint, so no requirement to do it again.